content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -101,6 +101,26 @@ datasource db {
 }
 ```
 
+### <inlinecode>datasourceUrl</inlinecode>
+
+Shorthand for programmatically overriding the [`datasource`](#datasources) property.
+
+#### Property
+
+| Option                     | Example value        | Description                                                                   |
+| -------------------------- | -------------------- | ----------------------------------------------------------------------------- |
+| Database connection string | `'file:./dev_qa.db'` | The database [connection URL](/reference/database-reference/connection-urls). |
+
+#### Examples
+
+```ts
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient({
+  datasourceUrl: 'file:./dev_qa.db',
```

Comment:
99% of our users want a connection string there, not a SQLite file path. Let's use that as the example.

Reply:
The file path is still a connection string, only that it's specific to SQLite, correct?

Reply:
Yes, but for 99% of our users it will look unexpected nad not how what they are trying to do and "scan" for.